manifests/init.pp

```diff
@@ -363,6 +375,12 @@
   Stdlib::Port $port                                             = 6379,
   Boolean $protected_mode                                        = true,
   Optional[String] $ppa_repo                                     = undef,
+  Boolean $redis_apt_repo                                        = false,
```

Comment:
please add the data here and not in params.pp. that enables puppet-strings to pick it up and put it into the reference.md